discord/state.py

```diff
@@ -463,7 +463,11 @@ def parse_message_reaction_add(self, data):
         if message is not None:
             emoji = self._upgrade_partial_emoji(emoji)
             reaction = message._add_reaction(data, emoji, raw.user_id)
-            user = self._get_reaction_user(message.channel, raw.user_id)
+            if raw.member:
+                user = Member(data=data, guild=message.channel.guild, state=self)
```

Comment:
I am 99% sure you are not supposed to pass `data`. Also  **that's after dispatching on_raw_reaction_add!**